template_feed/Microsoft.DotNet.Common.ProjectTemplates.1.x/Microsoft.DotNet.Common.ProjectTemplates.1.x.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>netstandard1.0</TargetFramework>
+        <IncludeBuildOutput>False</IncludeBuildOutput>
+        <IncludeSource>False</IncludeSource>
+        <GenerateAssemblyInfo>False</GenerateAssemblyInfo>
+        <OutputPath>$(TemplatesDir)</OutputPath>
+        <EnableDefaultItems>False</EnableDefaultItems>
+        <IsPackable>true</IsPackable>
+        <NoWarn>2008;NU5125</NoWarn>
+
+        <PackageId>Microsoft.DotNet.Common.ProjectTemplates.1.x</PackageId>
```

Comment:
What about the version and license ?

Reply:
Version is shared across all packages and is available under https://github.com/dotnet/templating/blob/master/eng/Versions.props

LicenceUrl is deprecated from NuGet. It seems there are some default licence properties from arcade. I will double check to make sure all the required metadata are provided for the packages. 

I also noticed that there are some additional contents in the packages. Will fix them too.

Reply:
The latest update should fix both these issues.